src/libraries/System.Data.OleDb/src/DbConnectionOptions.cs

```diff
@@ -228,7 +228,11 @@ internal static void AppendKeyValuePairBuilder(StringBuilder builder, string key
                     // <value> -> <value>
                     builder.Append(keyValue);
                 }
+#if NETCOREAPP
+                else if (keyValue.Contains('\"') && !keyValue.Contains('\''))
+#else
                 else if ((-1 != keyValue.IndexOf('\"')) && (-1 == keyValue.IndexOf('\'')))
+#endif
```

Comment:
I don't think this is worth ifdef'ing for.  I'd just suppress the warning somewhere, including in the csproj.